### PR TITLE
fix(layout): docs

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: libs/components/src/lib/layout/README.md [hunk 1]
```diff
@@ -41,6 +41,14 @@ Use the `gutters` attribute to set the initial preferred `margin`.
 - Default: `undefined`
 
 ```html preview full
+<style>
+  vwc-card {
+    width: 100%;
+  }
+  html { 
+    block-size: 400px; 
+  }
```
Comment (yinonov) [inline libs/components/src/lib/layout/README.md:50]: is this required?

File: libs/components/src/lib/layout/README.md [hunk 2]
```diff
@@ -65,6 +73,14 @@ Use the `column-spacing` attribute to set the initial preferred `spacing` of a c
 - Default: `'medium'`
 
 ```html preview full
+<style>
+  vwc-card {
+    width: 100%;
+  }
+  html { 
+    block-size: 300px; 
+  }
```
Comment (yinonov) [inline libs/components/src/lib/layout/README.md:82]: is this required?

File: libs/components/src/lib/layout/README.md [hunk 3]
```diff
@@ -135,6 +159,17 @@ With `fill`, everything is the same as `fit`, except empty tracks are not collap
 - Default: `'fit'`
 
 ```html preview full
+<style>
+  vwc-card {
+    width: 100%;
+  }
+  html { 
+    block-size: 200px; 
+  }
```
Comment (yinonov) [inline libs/components/src/lib/layout/README.md:168]: is this required?

File: libs/components/src/lib/layout/README.md [hunk 4]
```diff
@@ -93,6 +109,14 @@ Use `block` to stack elements one above the other.
 - Default: `'medium'`
 
 ```html preview full
+<style>
+  vwc-card {
+    width: 100%;
+  }
+  html { 
+    block-size: 1000px; 
+  }
```
Comment (yinonov) [inline libs/components/src/lib/layout/README.md:118]: is this required?

File: libs/components/src/lib/layout/README.md [hunk 5]
```diff
@@ -158,6 +193,17 @@ Controls the grid-template-columns of the layout.
 - Default: auto
 
 ```html preview full
+<style>
+  vwc-card {
+    width: 100%;
+  }
+  html { 
+    block-size: 200px; 
+  }
```
Comment (yinonov) [inline libs/components/src/lib/layout/README.md:202]: is this required?

File: libs/components/src/lib/layout/README.md [hunk 6]
```diff
@@ -41,6 +41,14 @@ Use the `gutters` attribute to set the initial preferred `margin`.
...
+  vwc-card {
+    width: 100%;
+  }
+  html { 
+    block-size: 400px; 
+  }
+</style>
 <div class="demo-gradient">
   <vwc-layout>
     <vwc-card elevation="2" text="default"></vwc-card>
```
Comment (yinonov) [inline libs/components/src/lib/layout/README.md:54]: adding redundant members, such as elevation here, only affect the looks and beautifies the docs but functionality-wise it reduces the readability and dev experience

**Review by yinonov — CHANGES_REQUESTED** [review]
as mentioned before, I believe the gradient div & class doesn't belong inside the exposed doc.

in the future we should enable that from another preview attribute

File: libs/components/src/lib/layout/README.md (hunk 1 above)
Comment (rinaok) [reply libs/components/src/lib/layout/README.md:50]: @yinonov yes, it doesn't look good without it

File: libs/components/src/lib/layout/README.md (hunk 6 above)
Comment (rachelbt) [reply libs/components/src/lib/layout/README.md:54]: it's not elevation its card inside layout. 
There's no need for layout if there are no components inside it :)

Comment (rinaok) [reply libs/components/src/lib/layout/README.md:54]: @yinonov  It was a div before and you requested that it be changed to a card component.

File: libs/components/src/lib/layout/README.md (hunk 2 above)
Comment (rinaok) [reply libs/components/src/lib/layout/README.md:82]: @yinonov yes, it doesn't look good without it

File: libs/components/src/lib/layout/README.md (hunk 3 above)
Comment (rinaok) [reply libs/components/src/lib/layout/README.md:168]: @yinonov yes, it doesn't look good without it

File: libs/components/src/lib/layout/README.md (hunk 5 above)
Comment (rinaok) [reply libs/components/src/lib/layout/README.md:202]: @yinonov yes, it doesn't look good without it

Comment (yinonov) [reply libs/components/src/lib/layout/README.md:202]: I don't get it. why we need to apply size to html here?

Comment (rinaok) [reply libs/components/src/lib/layout/README.md:202]: @yinonov What would be a better solution?

File: libs/components/src/lib/layout/README.md (hunk 1 above)
Comment (rinaok) [reply libs/components/src/lib/layout/README.md:50]: Screen Shot 2022-06-14 at 12 08 49

File: libs/components/src/lib/layout/README.md (hunk 2 above)
Comment (rinaok) [reply libs/components/src/lib/layout/README.md:82]: Screen Shot 2022-06-14 at 12 09 13

File: libs/components/src/lib/layout/README.md (hunk 3 above)
Comment (rinaok) [reply libs/components/src/lib/layout/README.md:168]: Screen Shot 2022-06-14 at 12 09 30

File: libs/components/src/lib/layout/README.md (hunk 4 above)
Comment (rinaok) [reply libs/components/src/lib/layout/README.md:118]: Screen Shot 2022-06-14 at 12 09 50

**Pull request closed without merging**